core/src/test/java/org/pentaho/metaverse/impl/model/kettle/json/TransMetaJsonDeserializerTest.java

```diff
@@ -134,6 +136,8 @@ public class TransMetaJsonDeserializerTest {
   @BeforeClass
   public static void init() throws KettleException {
     KettleEnvironment.init();
+    DefaultBowl.getInstance().setSharedObjectsIO( new MemorySharedObjectsIO() );
```

Comment:
The test class is not using this DefaultBowl. How is that used?

Reply:
The test uses `transMeta.getDatabases()`, which includes those from the bowl.